templates/boltzmann.js

```diff
@@ -57,6 +57,9 @@ const redis = require('handy-redis')
 // {% if postgres %}
 const pg = require('pg')
 // {% endif %}
+const addAJVFormats = ajv => (require('ajv-formats')(ajv), ajv)
+const addAJVKeywords = ajv => (require('ajv-keywords')(ajv), ajv)
```

Comment:
Introduce a slightly-goofy wrapper function because these packages export a function that mutates its argument without returning it.